plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/application/TemplateListPage.java

```diff
@@ -151,9 +154,15 @@ public void widgetSelected(SelectionEvent e) {
 						try {
 							model.setTemplateFileName(file);
 							return;
-						}catch(ResourceFactoryException | ClassCastException err) {
+						}catch(ClassCastException err) {
```

Comment:
why is the exception not passed to the error dialog so it will be shown in Details... button ?